spec/config/initializers/secure_headers_spec.rb

```diff
@@ -0,0 +1,14 @@
+RSpec.describe 'config.ssl_options' do
+  subject(:ssl_options) { Rails.application.config.ssl_options }
+
+  it 'is configured to use Strict-Transport-Security (HSTS)' do
+    basic_app = lambda { |env| [200, {}, []] }
+    ssl_middleware = ActionDispatch::SSL.new(basic_app, **ssl_options)
+
+    request = { 'HTTPS' => 'on' }
+    _status, headers, _body = ssl_middleware.call(request)
+
+    expect(headers['Strict-Transport-Security']).
+      to eq('max-age=31556952; includeSubDomains; preload')
```

Comment:
1. this is slightly different than the `31536000` is the old test, but I figured it was close enough
2. I create this new test here, since the SSL middleware is not added to tests by default, the old request specs didn't exercise it. I couldn't figure out a nice way to add the SSL middleware for just that test, so I went with this approach of just testing the middleware directly with our app's config